test/qa/test_meshnet.py

```diff
@@ -27,26 +34,15 @@ def setup_module(module):
     meshnet.add_peer(ssh_client)
 
 
-def teardown_module(module):
+def teardown_function(function):
+    logging.log(data=info.collect())
+    logging.log()
     meshnet.revoke_all_invites()
     meshnet.remove_all_peers()
-    ssh_client.exec_command("nordvpn set mesh off")
-    sh.nordvpn.set.meshnet.off()
-    ssh_client.exec_command("nordvpn logout --persist-token")
-    sh.nordvpn.logout("--persist-token")
+    ssh_client.exec_command("nordvpn set defaults")
+    sh.nordvpn.set.defaults()
     daemon.stop_peer(ssh_client)
     daemon.stop()
```

Comment:
In `setup_function` we are logging in before each test. Shouldn't we also log out on teardown after each test?

Reply:
So it is also not needed here? :point_down:
https://github.com/NordSecurity/nordvpn-linux/blob/main/test/qa/test_routing.py#L34
https://github.com/NordSecurity/nordvpn-linux/blob/main/test/qa/test_allowlist_port.py#L31
https://github.com/NordSecurity/nordvpn-linux/blob/main/test/qa/test_allowlist_subnet.py#L36

Reply:
Yes, not needed there as well, just double-checked :D